src/js/components/common/dialogs/physicaldeviceonboarding.js

```diff
@@ -12,10 +12,6 @@ import { getDebConfigurationCode } from '../../../helpers';
 import { advanceOnboarding } from '../../../utils/onboardingmanager';
 
 const types = [
-  {
-    title: 'BeagleBone',
-    value: 'beaglebone'
-  },
```

Comment:
I would also remove:

```
  {
    title: 'Generic ARMv6 or newer',
    value: 'generic-armv6'
  }
```

@estenberg, do you object?

Reasoning being, our onboarding use-cases (application, docker, system update) will only work on Raspberry Pi or QEMU and even if technically the "generic ARMv6" would support the "application" use-case it would become very confusing at the end when we link to the system & docker updates sections which only work on Raspberry Pi/QEMU (docker is an exception on QEMU but we are trying to fix that).

Before I comment on the rest of the rest (as this I think has impact on the rest of the content), lets sync up here :).

If we keep this we have 3 paths:

- Raspberry Pi - > application update -> system update -> docker update
- QEMU -> application update -> system update -> docker update (* not supported today but hopefully we can fix this)
- Generic ARM -> Application update

Reply:
I think you are right, @mirzak 
Let's remove the generic one from here. I will update the task & acceptance criteria.